cli/code_scanner.py

```diff
@@ -531,3 +552,58 @@ def _does_severity_match_severity_threshold(severity: str, severity_threshold: s
         return True
 
     return detection_severity_value >= Severity.try_get_value(severity_threshold)
+
+
+def _get_scan_result(cycode_client, scan_async_result: ScanInitializationResponse,
+                     scan_details: ScanDetailsResponse) -> ZippedFileScanResult:
+    scan_result = ZippedFileScanResult(did_detect=False, detections_per_file=[],
+                                       scan_id=scan_async_result.scan_id,
+                                       report_url=_try_get_report_url(scan_details.metadata))
+    if not scan_details.detections_count:
+        return scan_result
+
+    wait_for_detections_creation(cycode_client, scan_async_result.scan_id, scan_details.detections_count)
```

Comment:
what if you reach to the polling timeout, you still wants to continue and get the scan's detections?